### ISSUE TITLE
`ckan.site_description` isn't used

### ISSUE THREAD

**Issue by wood-chris** [ckan/ckan#6111]
**CKAN version**
2.9

**Describe the bug**
The docs (https://docs.ckan.org/en/2.9/maintaining/configuration.html#ckan-site-description) say:

> This is for a description, or tag line for the site, as displayed in the header of the CKAN web interface.

But when I set this on a brand new install of CKAN, the only place it's updated is in the "Site Tag Line" field in the config page (/ckan-admin/config) - I also note that this field doesn't have any help info in the left hand panel

**Expected behavior**
Either this option should be implemented somewhere, or the docs should explain that maintainers need to implement it themselves

**Comment by wood-chris** [comment]
Err - maybe I had caching issues...

**Comment by Zharktas** [comment]
Yup, description or tag line as it is labeled in configuration is only shown if site_logo is not defined.

https://github.com/ckan/ckan/blob/27f0b8cf52bb2140f16e2f09f3c42ed15e7e9b99/ckan/templates/header.html#L75-L82

`g.site_description` is in the else block, could be changed that it is shown also on with the logo. Needs of course checking that the layout still works with the logo too.

**Comment by Zharktas** [comment]
![image](https://user-images.githubusercontent.com/830663/119610240-950f8880-be01-11eb-9adf-e2451d2f96bf.png)
This is how it appears when there is no logo.

**Comment by Deep-NEC** [comment]
@Zharktas Did issue #6111 is similar to issue #7000?
